tests/specs/lockfile/no_lock/fail.out

```diff
@@ -1,4 +1,5 @@
 Download http://localhost:4545/lockfile/basic/mod.ts
+Download http://localhost:4545/lockfile/basic/mod.ts
```

Comment:
It redownloads files if they've never been downloaded. I think it's fine because it's a rare error case.